### DISCUSSION TITLE
build fails on AArch64, Fedora 33

### DISCUSSION THREAD

**Discussion by LutzWeischerFujitsu** [rabbitmq/rabbitmq-server#2913] [Other]
[jw@cn05 rabbitmq-server]$ make VERBOSE=1 -j$(proc)
-bash: proc: command not found
make[1]: Entering directory '/home/jw/rabbitmq-server/deps/rabbit_common'
Error: No Makefile to build dependency /home/jw/rabbitmq-server/deps/jsx.
make[1]: *** [erlang.mk:4512: deps] Error 2
make[1]: Leaving directory '/home/jw/rabbitmq-server/deps/rabbit_common'
make: *** [erlang.mk:4512: deps] Error 2
[jw@cn05 rabbitmq-server]$

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment]
This is very light on details about your build environment. We test packages on several Fedora versions, the latest right now being 32 IIRC. [This guide](https://www.rabbitmq.com/build-server.html) explains what build system dependencies the environment must provide.

**Comment by LutzWeischerFujitsu** [comment]
Retried, seems to build now without errors.

**Reply by mxmcarl** [reply]
> Retried, seems to build now without errors.
@LutzWeischerFujitsu 
Hello, I reported the same error when compiling. I installed the dependencies in the guide. I don't know what's missing
